fastlane/spec/actions_specs/setup_circle_ci_spec.rb

```diff
@@ -2,6 +2,21 @@
   describe Fastlane::Actions::SetupCircleCiAction do
     describe "Setup CircleCi Integration" do
       let(:tmp_keychain_name) { "fastlane_tmp_keychain" }
+      is_macos_env = RUBY_PLATFORM.downcase.include?("darwin")
```

Comment:
See above comment for making tests not dependent on platform being executed on ☝️